src/api/users/test/e2e/utils.js

```diff
@@ -0,0 +1,134 @@
+const fetch = require('node-fetch');
+const { hash, createServiceToken } = require('@senecacdot/satellite');
+
+const USERS_URL = 'http://localhost/v1/users';
+
+// takes a user object, performs GET using the user's email
+const getUser = async (user) => {
+  const response = await fetch(`${USERS_URL}/${hash(user.email)}`, {
+    headers: {
+      Authorization: `bearer ${createServiceToken()}`,
+      'Content-Type': 'application/json',
+    },
+  });
+
+  return response;
+};
+
+// constructs a paginated query using a passed string
+const getUsersPaginated = async (query = '') => {
+  const response = await fetch(`${USERS_URL}/${query}`, {
+    headers: {
+      Authorization: `bearer ${createServiceToken()}`,
+      'Content-Type': 'application/json',
+    },
+  });
+
+  return response;
+};
+
+// takes an array of users and sequentially gets them all
+// not to be confused with getting users via pagination
+const getUsers = async (users) => Promise.all(users.map((user) => getUser(user)));
+
+const createUsers = (numberOfUsers, sorted = false) => {
+  let users = [...Array(numberOfUsers).keys()].map((index) => {
+    return {
+      firstName: `TelescopeUser${index}`,
+      lastName: `TelescopeUser${index}`,
+      email: `TelescopeUser${index}@email.com`,
+      hash: hash(`TelescopeUser${index}@email.com`),
+      displayName: `TelescopeUser${index} TelescopeUser${index}`,
+      isAdmin: false,
+      isFlagged: false,
+      feeds: [`https://dev.to/feed/TelescopeUser${index}`],
+      github: {
+        username: `TelescopeUser${index}`,
+        avatarUrl:
+          'https://avatars.githubusercontent.com/u/7242003?s=460&u=733c50a2f50ba297ed30f6b5921a511c2f43bfee&v=4',
+      },
+    };
+  });
+
+  if (sorted) {
+    users.sort((a, b) => {
+      if (a.hash > b.hash) return 1;
+      if (a.hash < b.hash) return -1;
+      return 0;
+    });
+  }
+
+  return users.map((user) => {
+    delete user.hash;
+    return user;
+  });
+};
+
+const postUser = async (user) => {
+  const response = await fetch(`${USERS_URL}/${hash(user.email)}`, {
```

Comment:
Small thing, but you don't need to `await` these, since all you do is return the response:

```js
const postUser = (user) => fetch(...);
```

You don't need to change them all, but FYI for next time.